back/src/adapters/primary/scripts/triggerDeleteEmailAttachements.ts

```diff
@@ -0,0 +1,37 @@
+import { Pool } from "pg";
+import { createLogger } from "../../../utils/logger";
+import { PgNotificationRepository } from "../../secondary/pg/PgNotificationRepository";
+import { AppConfig } from "../config/appConfig";
+import { handleEndOfScriptNotification } from "./handleEndOfScriptNotification";
+
+const logger = createLogger(__filename);
+const config = AppConfig.createFromEnv();
+
+const executeTriggerDeleteEmailAttachements = async () => {
+  const dbUrl = config.pgImmersionDbUrl;
+  const pool = new Pool({
+    connectionString: dbUrl,
+  });
+  const client = await pool.connect();
+
+  const pgNotificationRepository = new PgNotificationRepository(client);
+  const numberOfUpdatedRows =
+    await pgNotificationRepository.deleteAllEmailAttachements();
+
+  client.release();
+  await pool.end();
+
+  return {
+    success: true,
+    numberOfUpdatedRows,
```

Comment:
si on arrive ici, on a forcément `success: true`...
dans ce cas, on peut juste retourner
```suggestion
    numberOfUpdatedRows,
```
ou
```suggestion
    numberOfAttachmentsDeleted,
```